mlflow/utils/databricks_utils.py

```diff
@@ -763,11 +827,43 @@ def _construct_databricks_model_version_url(
     return model_version_url
 
 
+def _get_databricks_creds_config(tracking_uri):
+    # Note:
+    # `_get_databricks_creds_config` reads credential token values or password and
+    # generates environmental variables.
```

Comment:
Does `generates` mean `set`?

Reply:
oh , comment is wrong. it doesn't set environment vars. just return a config object